scripts/create-package/plopfile.ts

```diff
@@ -137,47 +129,55 @@ module.exports = (plop: NodePlopAPI) => {
             return newPackageJson;
           },
         },
+        // update example package.json
+        // update package.json
         {
-          // update example package.json
-          // update package.json
           type: 'modify',
           path: `${exampleRoot}/package.json`,
           skip: () => {
-            if (!hasExamples) {
-              return 'Skipping react-examples package.json update';
-            }
+            if (!hasExamples) return 'Skipping react-examples package.json update';
           },
           transform: packageJsonContents => updateExamplePackageJson(packageJsonContents, data.packageNpmName),
         },
+        // update tsconfig.json
         {
-          // update tsconfig.json
           type: 'modify',
           path: `${destination}/tsconfig.json`,
           transform: tsconfigContents => updateTsconfig(tsconfigContents, hasTests),
         },
         () => {
           if (hasError) {
-            console.error(
+            console.log(
               chalk.red.bold(
-                '\nThere were one or more errors creating the package.\n' +
-                  'Please look at the logs above, fix the issues, and then run `yarn` to link.',
+                stripIndent(`
```

Comment:
Same comments as earlier about `stripIndent`. I'd prefer to just get rid of it and go back to the previous approach in both this case and the ones below.